### DISCUSSION TITLE
Cookbook rules

### DISCUSSION THREAD

**Discussion by staubzuechterin** [mealie-recipes/mealie#4307] [Q&A] [answered]
I'm in the process of filling my Mealie (selfhosted, v1.2.0) and I'm running into a problem with the cookbooks.

I have a category for baby food, which only contains dishes for babies. But there are also recipes in other categories that can be made suitable for babies with a little extra effort. These dishes have a corresponding tag (“Baby food ideas”), but are not in the category of baby food, because they are still normal dishes. 

Now I have created a cookbook with the category baby food and the tag “baby food ideas”. All filter options of the cookbook are deactivated. Now I actually expect all recipes from the baby food category and all recipes with the tag “Baby food ideas” to appear, but the cookbook remains empty. Since these rules are not an “OR” but apparently an “AND”?... or am I doing something wrong?

**Comment by michael-genson** [comment, marked as answer]
If you provide any categories, the recipe must have one of those categories. Similarly, if you provide any tags, the recipe must have one of those tags. Currently we don't support cookbooks for "one of these categories OR one of these tags".

I think supporting this makes sense, it's just not currently supported that way.

**Reply by staubzuechterin** [reply]
Thanks!

Then I will probably tag all baby food recipes with the tag for the time being and adapt the rules until it will perhaps be possible to link the rules with “OR” at some point ;)
